demos/motmetrics4norfair/src/motmetrics4norfair.py

```diff
@@ -114,35 +95,39 @@ def keypoints_distance(detected_pose, tracked_pose):
             input_path=input_path, save_path=output_path, information_file=info_file
         )
 
-    if args.make_video:
-        video_file = video.VideoFromFrames(
-            input_path=input_path, save_path=output_path, information_file=info_file
-        )
+    video_file = video.VideoFromFrames(
+        input_path=input_path,
+        save_path=output_path,
+        information_file=info_file,
+        make_video=args.make_video,
+    )
 
     tracker = Tracker(
-        distance_function=keypoints_distance,
-        distance_threshold=distance_threshold,
-        detection_threshold=detection_threshold,
-        pointwise_hit_counter_max=pointwise_hit_counter_max,
-        hit_counter_max=hit_counter_max,
+        distance_function=iou,
+        distance_threshold=DISTANCE_THRESHOLD,
+        detection_threshold=DETECTION_THRESHOLD,
+        pointwise_hit_counter_max=POINTWISE_HIT_COUNTER_MAX,
+        hit_counter_max=HIT_COUNTER_MAX,
         filter_factory=FilterPyKalmanFilterFactory(),
```

Comment:
why are we still using filterpy? OptimizedFilter is faster and it _shouldn't_ affect metrics too much.

Reply:
I tried using `OptimizedKalmanFilterFactory`, but the metrics were a bit worse. We saw this when implementing `OptimizedKalmanFilterFactory` but we still haven't gotten to the bottom of it.